### PR TITLE
Overhaul readme & repo structure

### PR DESCRIPTION
**Pull request by axe312ger**
Goals:

1. Make readme & docs easier to digest
2. Lower contribution barrier
3. Most important: Add emojis!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,44 @@
...
+<p align="center">
+    <a href="https://www.contentful.com/slack/"><img src="https://img.shields.io/badge/-Join%20Community%20Slack-2AB27B.svg?logo=slack&maxAge=31557600" alt="Join Contentful Community Slack"></a>
+    <a href="https://www.contentfulcommunity.com/"><img src="https://img.shields.io/badge/-Join%20Community%20Forum-3AB2E6.svg?logo=data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCA1MiA1OSI+CiAgPHBhdGggZmlsbD0iI0Y4RTQxOCIgZD0iTTE4IDQxYTE2IDE2IDAgMCAxIDAtMjMgNiA2IDAgMCAwLTktOSAyOSAyOSAwIDAgMCAwIDQxIDYgNiAwIDEgMCA5LTkiIG1hc2s9InVybCgjYikiLz4KICA8cGF0aCBmaWxsPSIjNTZBRUQyIiBkPSJNMTggMThhMTYgMTYgMCAwIDEgMjMgMCA2IDYgMCAxIDAgOS05QTI5IDI5IDAgMCAwIDkgOWE2IDYgMCAwIDAgOSA5Ii8+CiAgPHBhdGggZmlsbD0iI0UwNTM0RSIgZD0iTTQxIDQxYTE2IDE2IDAgMCAxLTIzIDAgNiA2IDAgMSAwLTkgOSAyOSAyOSAwIDAgMCA0MSAwIDYgNiAwIDAgMC05LTkiLz4KICA8cGF0aCBmaWxsPSIjMUQ3OEE0IiBkPSJNMTggMThhNiA2IDAgMSAxLTktOSA2IDYgMCAwIDEgOSA5Ii8+CiAgPHBhdGggZmlsbD0iI0JFNDMzQiIgZD0iTTE4IDUwYTYgNiAwIDEgMS05LTkgNiA2IDAgMCAxIDkgOSIvPgo8L3N2Zz4K&maxAge=31557600" alt="Join Contentful Community Forum"></a>
+    <a href="https://contentful.github.io/contentful.js">
```
Comment (dborsatto) [inline README.md]: I honestly think this is _way too much_ 😅

Comment (axe312ger) [reply README.md]: No, MOAR BADGES! 

Ofc not, that was just a test. This is just a in between step, contact me in slack if you want to learn more about the current state :)

File: README.md [hunk 2]
```diff
@@ -46,36 +46,33 @@
...
-  - [📜 License](#📜-license)
-  - [🗣 Code of Conduct](#🗣-code-of-conduct)
-
-<!-- /TOC -->
+
+- [🚀 Core Features](#core-features)
+  - [Supported browsers and Node.js versions:](#supported-browsers-and-nodejs-versions)
+- [🛫 Getting started](#getting-started)
+  - [☁️ Installation](#☁installation)
+    - [Using it directly in the browser:](#using-it-directly-in-the-browser)
```
Comment (Khaledgarbaya) [inline README.md]: U no emoji !

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -1,25 +1,69 @@
...
+
+Node Version: output of `node -v`
+Npm Version: output of `npm -v`
+Operating System: output of `uname -a` (UNIX), or version and 32 or 64-bit (Windows)
+Package Version: output of `npm list [package-name]`
+-->
 
-<!-- Enter your issue details below this comment. -->
+-   **Language Version**:
+-   **Package Manager Version**:
```
Comment (Khaledgarbaya) [inline .github/ISSUE_TEMPLATE.md]: can you add instruction to pull these data: like `npm --version`

**Pull request merged**